app/graphql/mutation/createTrip.graphql

```diff
@@ -0,0 +1,24 @@
+mutation createTrip(
```

Comment:
I have a question. If you follow feature directory, this schema file also should be put under feature directory? For example, `app/trip/graphql/mutation/createTrip.graphql`
Or about graphql, we should gather under this directory? The query file is same thing. If the query/mutation will be used across several features,  maybe we can put under this directory? What do you think? 👀

Reply:
Good idea about the `provider` directory! But `Provider.tsx` is just a renamed `chakra-provider.tsx`.
Maybe we can just rename it for clarity instead? What do you think? 😊

Reply:
You're spot on with the idea of organizing GraphQL files! However, since we don't have a trip directory yet, I've placed the GraphQL files directly under the app directory for now. It's a temporary setup until we decide on a more permanent structure. Does this sound good for the time being? 🙏

Reply:
Ok! You can leave it for now, and I will move it when I make graphql for top page :)